esx_service/cli/vmdkops_admin.py

```diff
@@ -1251,74 +1250,63 @@ def config_init(args):
     if not args.local:
         print("Creating a symlink to DB at {}".format(link_path))
         create_db_symlink(db_path, link_path)
-    service_restart()
-    return None
+
+    return service_reset()
 
 
 def config_rm(args):
     """
-    Remove Config DB.BaseException
+    Remove Local Config DB or local link.
     :return: None for success, string for error
     """
 
-    # TODO
-    # This command should ask for double confirmation, reset the DB_PATH,
-    # and delete the database instance
-    # Other ESXs will detect this change (DB is missing) and
-    # reset the DB_PATH (if new DB cannot be discovered)
+    # This asks for double confirmation, and removes the local link or DB (if any)
+    # NEVER deletes the shared database - instead prints help
 
-    err = check_ds_local_args(args)
-    if err:
-        return err
-
-    if not args.force:
-        return err_out("Warning: Config DB removal requires '--force' flag to execute the request")
-
-    if args.datastore:
-        db_path = auth_data.AuthorizationDataManager.ds_to_db_path(args.datastore)
+    if not args.local:
+        return err_out("""
+        Shared DB removal is not supported. For removing  local configuration, use --local flag.
```

Comment:
Ideally it would be nice to dump list of ESX still participating in Tenancy so that Admin don't have to remember or check every ESX host. File a issue for later?